0.12.1 release (#422)
packit-service/ogr: CHANGELOG.md

# 0.12.1

## New Features

- PullRequest can now be created also between two different forks of
  a project. (@mfocko)

> - Pull requests have now property - patch (Pagure). (@jpopelka)

Reviewer: ```suggestion
- PullRequests have a `patch` property now when working with Pagure. (@jpopelka)
```

> - You can now add collaborators with specific privileges on github and
>   gitlab projects. (@shreyaspapi)

Reviewer 2: ```suggestion
- You can now add collaborators with specific privileges on GitHub and
  GitLab projects. (@shreyaspapi)
```

> - You can now add collaborators with specific privileges on github and
>   gitlab projects. (@shreyaspapi)

Reviewer: ```suggestion
- You can now add collaborators with specific privileges on GitHub and
  GitLab projects. (@shreyaspapi)
```

## Minor

- When telling if a user can merge PRs in a GitHub repo, ogr asks GitHub
  for the user's permission on the repo instead of checking if the user is
  in the list of collaborators. (@csomh)

## Fixes

> - Get project will now correctly use the service instance class for custom

Reviewer: ```suggestion
- `get_project()` will now correctly use the service instance class for custom.
```

> - Get project will now correctly use the service instance class for custom
>   service instances. (@lachmanfrantisek)

Reviewer 2: ```suggestion
- factory.get_project() will now correctly use service instance class for custom
  service instances. (@lachmanfrantisek)
```
